src/helpers/useTransactionHandler.tsx

```diff
@@ -134,18 +139,10 @@ export const useTransactionHandler = ({
               txHash: undefined,
               loading: false,
             });
-
-            // set use permit to false to retry with normal approval
-            setUsePermit(false);
```

Comment:
I think there's some cleanup we could do in here now. I don't think we need `usePermit` as state and we can just use the param passed in. It's also being returned from this hook, but doesn't look like it's used anywhere.